Community/Tdarr_Plugin_henk_Keep_Native_Lang_Plus_Eng.js

```diff
@@ -108,9 +108,15 @@ const processStreams = (result, file, user_langs) => {
   };
   let streamIndex = 0;
 
-  const langsTemp = result.original_language;
+  var langsTemp = result.original_language; //Change from const to var to re-define
+  
+  if (langsTemp == 'cn') langsTemp = 'zh'; //If the original language is pulled as Chinese 'cn' is used.  iso-language expects 'zh' for Chinese.
+  
   let langs = [];
 
+  /*
```

Comment:
Instead of commenting it out you can just remove it. Apparently I got `original_language` mixed up with `spoken_languages` somewhere in development. https://developers.themoviedb.org/3/movies/get-movie-details

Reply:
Did not know if you wanted to keep the loop so thus the comment out.